source/assets/stylesheets/styles/_project_story_large_tile.scss

```diff
@@ -0,0 +1,95 @@
+.project-story-large-tile {
+  box-sizing: border-box;
+  display: block;
+  margin-bottom: 6px;
+  padding-right: 6px;
+  text-decoration: none;
+
+  @include breakpoint($tablet-breakpoint) {
+    float: left;
+    margin-bottom: 0;
+    width: 66%;
+  }
+}
+
+.project-story-large-tile__container {
+  background-color: $project-story-tile__background-color;
+  box-sizing: border-box;
+  border: 1px solid $project-story-tile__border-color;
+  padding: 6px;
+  width: 100%;
+
+  &:hover {
+    background-color: $project-story-tile__hover-color;
+  }
+}
+
+.project-story-large-tile__image-container {
+  box-sizing: border-box;
+}
+
+.project-story-large-tile__image {
+  display: block;
+  height: 125px;
+  object-fit: cover;
+  width: 100%;
+
+  @include breakpoint($max-mobile-container) {
+    height: 180px;
+  }
+
+  @include breakpoint($tablet-breakpoint) {
+    height: 255px;
+  }
+
+  @include breakpoint($tablet-breakpoint) {
+    height: 276px;
+  }
+}
+
+.project-story-large-tile__content {
+  display: inline-block;
+  min-height: 60px;
+  padding-left: 8px;
+  padding-top: 10px;
+  text-decoration: none;
+
+  @include breakpoint($max-mobile-container) {
+    min-height: 75px;
+  }
+
+  @include breakpoint($tablet-breakpoint) {
+    padding: 20px 6px 0;
+    width: 85%;
+    min-height: 170px;
```

Comment:
Oh yeah, I wanted to make all tiles to be have same height. But you're right - it looks better when text determines it rather then it's fixed.